stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -0,0 +1,19 @@
+# oidc-authservice for IBM Cloud AppID
+
+## Prerequisites
+
+* Provisioning an AppID instance from IBM Cloud. the free tier works.
+* keep the configuration parameters including:
+    - client_id
+    - OIDC provider URL
+    - client_secret
+
+## How to use
+
+After provisioning the statefulset `authservice` in namespace `istio-system`,
```

Comment:
Should be clarified that the `authservice` is provisioned as part of the Kubeflow deployment. And it will always help to have a script to do this.

Reply:
It can be a kpt function or just a k8s secret, so far, I will just document it.